optimism.tokenlist.json

```diff
@@ -290,6 +290,14 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 10,
+      "address": "0xc7b04dC5A2644522a0C58c107346B5e3A63600B0",
+      "name": "Sarcophagus",
+      "symbol": "SARCO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/SARCO.png"
+    },
```

Comment:
You need to also add an entry for chainId 1, i.e. mainnet so to map the token from L1 to L2.